tests/Acceptance/Exercício6Cest.php

```diff
@@ -0,0 +1,123 @@
+<?php
+
+
+namespace Tests\Acceptance;
+
+use Tests\Support\AcceptanceTester;
+
+class Exercício6Cest
+{
+    public function _before(AcceptanceTester $I)
+    {
+    }
+
+    // tests
+    public function ÍndiceExercicio6Test(AcceptanceTester $I)
```

Comment:
Me parecer que  a letra I do início no nome do seu metodo está com acento agudo. 

Procure evitar letras com acentos em nomes de métodos, funções, variáveis.... pois isso pode causar erros na execução do código ou até mesmo erros de digitação.

Sugiro renomear este método de `ÍndiceExercicio6Test` para  `IndiceExercicio6Test`